spot/README.md

```diff
@@ -0,0 +1,16 @@
+# spot
+
+This folder contains some resources on writing *reentrant* model training scripts. Training scripts with this property are perfect for performing long-running modle training jobs using spot instances (aka preemptible instances on GCP) and/or on-demand instances on GCP.
+
+## Automating GPU machine failure recovery in Google Compute Engine
+
+![](https://i.imgur.com/NYowq6j.png)
+
+This blog post, ["Automating GPU machine failure recovery in Google Compute Engine"](https://spell.run/blog/automated-machine-failure-recovery-Xp3TEhEAACUAYwPM), discusses GPU host maintenance (aka the `REPAIRING` machine state) on GCP. The `reentrancy-demo.ipynb` Jupyter notebook contains some helpful code samples. You can launch this notebook from a Spell workspace by running the following CLI command (requires having the `spell` package installed):
```

Comment:
nit I'd remove `from a Spell workspace` part, sounds a bit confusing, even though i know they have to open the ipynb file after launching